docs/src/main/paradox/elasticsearch.md

```diff
@@ -215,3 +215,6 @@ Scala
 
 Java
 : @@snip [snip](/elasticsearch/src/test/java/docs/javadsl/ElasticsearchTest.java) { #custom-search-params }
+
+Additionally, support for custom routing is available through the `routing` key. 
```

Comment:
Might be worth adding a link to somewhere in the elasticsearch docs that explain some of the benefits of using custom routing if possible?